config/config.go

```diff
@@ -32,6 +43,13 @@ type Kafka struct {
 	AllowAutoTopicCreation      bool              `yaml:"allowAutoTopicCreation"`
 }
 
+func (k *Kafka) GetBalancer() kafka.Balancer {
+	if balancer, ok := balancerMap[k.Balancer]; ok {
+		return balancer
+	}
+	panic("Invalid kafka balancer method")
```

Comment:
We can also write k.Balancer here to be able to see.

Reply:
Done